src/common/components/paymentMethods/deletePaymentMethod/deletePaymentMethod.modal.tpl.html

```diff
@@ -0,0 +1,164 @@
+<div class="account-management">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-12">
+          <div class="border-bottom-small">
+            <button type="button" class="close" aria-label="Close" ng-click="$ctrl.dismiss()">
+              <span aria-hidden="true">×</span>
+            </button>
+            <h3 ng-if="$ctrl.view != 'addPaymentMethod'" class="text-center">Delete Payment Method</h3>
+            <h3 ng-if="$ctrl.view == 'addPaymentMethod'" class="text-center">Add Payment Method</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body pt0">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-md-12 col-xs-12">
+          <form>
+
+            <div ng-if="$ctrl.view != 'addPaymentMethod'" class="panel panel-default bg-danger">
+              <div class="panel-body">
+                <p>
+                  <span class="payment-icon">
```

Comment:
Add <div class="radio-method"></div> around the payment-method-display to fix that large image issue. I ran into the same thing in the your-giving component.